# Comment else statement to prevent use of non-existent directory
# See https://bugs.mysql.com/bug.php?id=87348
diff -ru mysql-5.7.21.orig/CMakeLists.txt mysql-5.7.21/CMakeLists.txt
--- mysql-5.7.21.orig/CMakeLists.txt	2018-10-25 13:54:46.583397887 -0500
+++ mysql-5.7.21/CMakeLists.txt	2018-10-25 13:57:19.307397723 -0500
@@ -637,9 +637,9 @@
   SET (MYSQLD_STATIC_EMBEDDED_PLUGIN_LIBS "" CACHE INTERNAL "")
   # Add storage engines and plugins.
   CONFIGURE_PLUGINS()
-ELSE()
+#ELSE()
   # We may still want Cluster client libraries, use -DWITH_NDBCLUSTER=1
-  ADD_SUBDIRECTORY(storage/ndb)
+#  ADD_SUBDIRECTORY(storage/ndb)
 ENDIF()
 
 IF(WITH_UNIT_TESTS)
